### DISCUSSION TITLE
Zoom not change + Shortkeys Problem

### DISCUSSION THREAD

**Discussion by mrusmandaud** [d2phap/ImageGlass#776] [General]
Zoom not changing means I want to change zoom level ( , ) comma not typing.
Short not changing. Custom

**Comment by d2phap** [comment]
Please use the template when creating an issue.
I am not sure I can understand what you try to say, but some info may be helpful to you:

In v7.6
- Zoom levels must be separated by `;` not `,`
- Zoom modes shortcuts are numbers from 1 to 6, you can see more in the main menu of ImageGlass

**Comment by mrusmandaud** [comment]
Yes it separating with ; not , this.
but not exceeding 3500% above not like 4000
